R/environment.R

```diff
@@ -199,14 +208,33 @@ container_registry <- function(address = NULL,
 #' @param github_packages character vector of github packages to be installed.
 #' @param custom_url_packages character vector of packages to be installed from
 #' local, directory or custom url.
+#' @param install_extra_packages logical parameter to specify if extra packages
+#' should be installed at runtime.
 generate_docker_file <- function(custom_docker_image = NULL,
                                  cran_packages = NULL,
                                  github_packages = NULL,
-                                 custom_url_packages = NULL) {
+                                 custom_url_packages = NULL,
+                                 install_extra_packages = TRUE) {
```

Comment:
would it be better to call it `system_managed_packages`?

Reply:
I renamed it to `install_system_packages`.